app/controllers/system_admin/user_lead_partners_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module SystemAdmin
+  class UserLeadPartnersController < ApplicationController
+    before_action { require_feature_flag(:lead_partners) }
+
+    before_action :set_user
+    helper_method :query
+
+    def index
+      @lead_partner_form = UserLeadPartnersForm.new
+      @lead_partner_search = PartnerSearch.call(query: query, lead_partners_only: true)
```

Comment:
Is this class to be implemented? I can see only [LeadPartnerSearch](https://github.com/DFE-Digital/register-trainee-teachers/blob/7168-add-lead-partner-users-in-support-console/app/services/lead_partner_search.rb#L3)